pkg/cmd/cli/cmd/newapp.go

```diff
@@ -132,11 +132,60 @@ type NewAppOptions struct {
 	LogsForObject LogsForObjectFunc
 }
 
+type NewAppOptions struct {
+	AppOptions *BaseOptions
```

Comment:
If you include it without a name, then you don't need to prefix all access with AppOptions, or use a separate variable (`ao`) like you do below.